.github/workflows/tox-pytest.yml

```diff
@@ -26,7 +26,6 @@ jobs:
           condarc: |
             channels:
             - conda-forge
-            - defaults
```

Comment:
conda / mamba recommends not including the default anaconda channel for package consistency.